specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -572,7 +637,7 @@
         },
         "properties": {
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/MetricAlertProperties",
+          "$ref": "#/definitions/MetricAlertPropertiesPatch",
```

Comment:
why do we change this one?

Reply:
I've added MetricAlertPropertiesPatch object to be similiar to MetricAlertProperties, but not required any field, as it's a Patch request.